index.d.ts

```diff
@@ -12,41 +12,41 @@ import { IncomingMessage, ServerResponse } from 'http';
 import pino from 'pino';
 import { err, req, res, SerializedError, SerializedRequest, SerializedResponse } from 'pino-std-serializers';
 
-declare function PinoHttp(opts?: Options, stream?: pino.DestinationStream): HttpLogger;
+declare function PinoHttp<IM = IncomingMessage, SR = ServerResponse>(opts?: Options<IM, SR>, stream?: pino.DestinationStream): HttpLogger<IM, SR>;
```

Comment:
Originally had all of these as `<Request = IncomingMessage, Response = ServerResponse>` but that added a **ton** of visual clutter to this file. I'm happy to revise these though if you'd like.